<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>832</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>832</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>832</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>602</width>
     <height>424</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color: white;
border-radius: 20px;
}</string>
   </property>
   <widget class="QLabel" name="firstName_LA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(103, 103, 103);
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName_LE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
border: 2px solid rgb(103,103,103);
border-radius:5px;
color: rgb(103, 103, 103);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="register_PB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(23,24,30);
border-radius:5px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="registration_LA">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="birth_LA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: transparent;
color:black;
}</string>
    </property>
    <property name="text">
     <string>Date of birth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName_LE">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
border: 2px solid rgb(103,103,103);
border-radius:5px;
color: rgb(103, 103, 103);

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lastName_LA">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(103, 103, 103);
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_LE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
border: 2px solid rgb(103,103,103);
border-radius:5px;
color: rgb(103, 103, 103);

}</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_LA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(103, 103, 103);
}</string>
    </property>
    <property name="text">
     <string>Email address</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_LA">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(103, 103, 103);
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_LE">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
border: 2px solid rgb(103,103,103);
border-radius:5px;
color: rgb(103, 103, 103);

}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="birthMonth_CB">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	border: 2px solid rgb(103,103,103);
	border-radius:7px;
	color: rgb(103,103,103);
}

QComboBox::down-arrow
{
image: url(:/Resources/Images/icons/Expand Arrow.png);
width:50px;
height:50px;

}

QComboBox::down-arrow::on
{
	image: url(:/Resources/Images/icons/Chevron Up.png);
	width:50px;
	height:50px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>Month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="birthDay_CB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	border: 2px solid rgb(103,103,103);
	border-radius:7px;
	color: rgb(103,103,103);
}

QComboBox::down-arrow
{
image: url(:/Resources/Images/icons/Expand Arrow.png);
}

QComboBox::down-arrow::on
{
	image: url(:/Resources/Images/icons/Chevron Up.png);
	width:50px;
	height:50px;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>Month</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="birthYear_CB">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	border: 2px solid rgb(103,103,103);
	border-radius:7px;
	color: rgb(103,103,103);
}

QComboBox::down-arrow
{
image: url(:/Resources/Images/icons/Expand Arrow.png);
width:50px;
height:50px;

}

QComboBox::down-arrow::on
{
	image: url(:/Resources/Images/icons/Chevron Up.png);
	width:50px;
	height:50px;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="calendarIcon_LA">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Resources/Images/icons/calendar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
